### ISSUE TITLE
Keyboard inputs are not visible

### ISSUE THREAD

**Issue by ssahaxd** [cantino/mcfly#247]
Keyboard inputs are not visible on the terminal after interrupting a command. 

![image](https://user-images.githubusercontent.com/11327529/163119378-e26beb87-266c-49ac-bda2-e0d1b844c226.png)
After interrupting the first command I typed this which was not visible on the terminal.

**Comment by cantino** [comment]
I think this has nothing to do with mcfly, but try typing `reset` and pressing return.

**Comment by MarceloLimoriGM2dev** [comment]
There's definitely something wrong. Here's a bash script that, if interrupted with Ctrl+C will leave you with a non-moving cursor, the "reset" command is needed to bring back the cursor. Issue doesn't happen if running the script directly without Mcfly.

```
#!/bin/bash
echo "Sleeping..."
sleep 10
echo "finished"
```

My system is bash under Linux Mint, btw.

**Comment by cantino** [comment]
I just tried this on zsh an bash on my Mac and it was fine. I'm not denying that there's an issue in your environment, though. I assume the same thing does not happen if you hit tab instead of return, then run the command?

**Comment by MarceloLimoriGM2dev** [comment]
Pressing tab seems to cause the same issue. 

I just discovered, I don't need to type the name of the script either, not even run it from Mcfly. I can just trigger Mcfly with Ctr-R, then press Esc (or tab) to exit, and then if and only if, the first thing I run is the script and I cancel it, then the issue happens. If I run something else first, then I can run the script and cancel it and no problem happens.

So to summarize:

Trigger Mcfly and exit.

1) Run script and cancel => issue.
2) Run script and not cancel => no issue.
3) Run something else and then run script and cancel => issue.

Let me know if there's some logging info I can fetch for you and thanks for your response.

**Comment by silviomarazzato** [comment]
Same issue:
Fedora 35
Mcfly 0.61
Tilix
Bash shell

I press CTRL-R and search for the command "yarn dev --https --port=8000" then press ENTER.
I interrupt the above command with CTRL-C and then I don't see output on the shell. I'm forced to type the "reset" command blindly to obtain newly the control.

**Comment by silviomarazzato** [comment]
My bash version is 5.1.8. I am seeing that 5.2.0 is still in beta and has not yet been released in the repositories of my distro. Is there any workaround to this problem?

**Comment by cantino** [comment]
I'm not aware of any workaround unfortunately, beyond typing 'reset' when it happens. If you find one, please share it!
